### PR TITLE
Spelling fixes & standard formatting using VS Code defaults

### PR DESCRIPTION
**Pull request by towner-10**
Standard formatting using the following `clang-format` settings:

```text
UseTab: (VS Code current setting)
IndentWidth: (VS Code current setting)
BreakBeforeBraces: Allman
AllowShortIfStatementsOnASingleLine: false
IndentCaseLabels: false
ColumnLimit: 0
```

### CODE REVIEW

**Comment by yamaan93** [comment]
what's your beef with `#pragma once`

**Comment by towner-10** [comment]
> what's your beef with `#pragma once`

You told me not to use it 💀

File: include/interfaces/pedal.h [hunk 1]
```diff
@@ -13,15 +13,15 @@ class Pedal
...
 
-        FS_CAN::CAN_SIGNAL sensorPlausabilitySignal{&sensorPlausability, 0,8, false, 1.0f,0};
-        FS_CAN::CAN_MSG pedalInfoMessage{400,{&sensorPlausabilitySignal}};
+        FS_CAN::CAN_SIGNAL sensorPlausibilitySignal{&sensorPlausibility, 0,8, false, 1.0f,0};
+        FS_CAN::CAN_MSG pedalInfoMessage{400,{&sensorPlausibilitySignal}};
 
         Sensor *sensor1;
         Sensor *sensor2;
 
-        float plausabilityFault; // float so we can Make it a CAN signal
```
Comment (yamaan93) [inline include/interfaces/pedal.h:24]: what did you even change here

File: include/lowPass.h
Comment (yamaan93) [inline include/lowPass.h:1]: Why did you rename this file?

File: include/interfaces/pedal.h (hunk 1 above)
Comment (yamaan93) [inline include/interfaces/pedal.h:24]: what did you even change here

**Review by yamaan93 — CHANGES_REQUESTED** [review]

**Comment by yamaan93** [comment]
> > what's your beef with `#pragma once`
> 
> You told me not to use it 💀

oh yeah true, it's technically not supported by all compilers i guess

File: include/interfaces/pedal.h (hunk 1 above)
Comment (towner-10) [reply include/interfaces/pedal.h:24]: I fixed the spelling.

File: include/lowPass.h
Comment (towner-10) [reply include/lowPass.h:1]: Not following the same convention we have used for other files using snake case.

Comment (yamaan93) [reply include/lowPass.h:1]: did you make sure it didn't break when you changed this

**Review by yamaan93 — APPROVED** [review]
